docs/metadata.md

```diff
@@ -76,9 +72,9 @@ ex. disease, vaccinations
 
 For several attributes, it does not make semantic sense to only allow one value for each metadata row. While we expect that in most controlled studies, these values will contain a single entry, we chose to allow multiple values for studies investigating combinitorial effects. Even if one of the values is not of specific interest to the study, please include it if possible. Ex. a study about flu vaccines would include the flu vaccine as a value for the vaccination attribute, but if the clinical data about other vaccines is also available, these vaccinations should be included as well. 
 
-**Syntax in metadata file:** Array-valued metadata should be formatted as follows: `["value1""value2""value3"]` or `"value1""value2""value3"`. If the array-valued metadata only contains a single value it should still follow the same format: ex `"value1"` or `["value1"]`.
+**Syntax in metadata file:** Array-valued metadata should be delimited using the pipe symbol (|): `[value1|value2|value3"]`. If the array-valued metadata only contains a single value, no special formatting is necessary.
 
-If the array-valued metadata attribute is a dependant metadata attribute, the order in the array should correspond with the order of the array the attribute depends on. For example, the following metadata entries for `vaccination__ontology_label`: `"Influenza Virus Vaccine""BCG Vaccine"`, for `vaccination__time_since` : `"1","2"`, and for `vaccination__time_since__unit_label`: `"year","year"` would indicate that the Influenza vaccine was recieved 1 year ago and the BCG Vaccine was recieved 2 years ago.  If information is not available for one of the metadata entries in a dependant array, include an empty string(`""`) for that value.
+If the array-valued metadata attribute is a dependant metadata attribute, the order in the array should correspond with the order of the array the attribute depends on. For example, the following metadata entries for `vaccination__ontology_label`: `Influenza Virus Vaccine|BCG Vaccine`, for `vaccination__time_since` : `1|2`, and for `vaccination__time_since__unit_label`: `year|year` would indicate that the Influenza vaccine was recieved 1 year ago and the BCG Vaccine was recieved 2 years ago.  If information is not available for one of the metadata entries in a dependant array, include a pipe symbol (`|`) at the appropriate location to indicate the lack of data. Example: `vaccination__time_since` : `|2`.
```

Comment:
Had not thought about missing data in arrays so "If information is not available for one of the metadata entries in a dependant array, include a pipe symbol (`|`) at the appropriate location to indicate the lack of data" needs to be tested to see if it will work as currently described.